src/main/java/com/kainos/ea/model/Job_Role.java

```diff
@@ -0,0 +1,41 @@
+package com.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Job_Role {
```

Comment:
Class name needs renamed to suit java naming conventions

Reply:
this is done